pkg/kapp/app/recorded_app.go

```diff
@@ -23,8 +23,10 @@ const (
 )
 
 type RecordedApp struct {
-	name   string
-	nsName string
+	name              string
+	nsName            string
+	creationTimestamp time.Time
+	labels            map[string]string
```

Comment:
We are trying to add more values to RecordedApp in another [PR](https://github.com/vmware-tanzu/carvel-kapp/pull/403/files#diff-c4cadf071a66c0e934e58c35eb0a9116fee4c744abd14621dc1eeebe7f5e9fb8R31-R32).
I feel like this might get out of hand soon. Should we do some grouping on these resources, something like RecordedAppMeta? (Absolutely open to other suggestions)

Reply:
Yeah make sense we can have RecordedAppMeta as a separate struct.

Reply:
Done

Reply:
We can put this on hold as lot of refactoring is required and get everyones view on this and then let's take a call on this ?

Reply:
I agree, If we want to do it, let's open another issue and work on it.